README.md

```diff
@@ -70,8 +70,10 @@ make test-with-coverage
 
 ### Commits
 
-This project uses [semantic versioning](https://semver.org/) and [conventional
-commits](https://www.conventionalcommits.org/en/v1.0.0/).
+This project currently uses loose [semantic versioning](https://semver.org/) and
```

Comment:
If the intent is to use strict semantic versioning once the project reaches 1.0.0, I'd suggest adding that comment here so folks know what to expect.